hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/BlacklistEndpoint.java

```diff
@@ -7,6 +7,7 @@
 import pl.allegro.tech.hermes.api.BlacklistStatus;
 import pl.allegro.tech.hermes.management.api.auth.Roles;
 import pl.allegro.tech.hermes.management.domain.blacklist.TopicBlacklistService;
+import pl.allegro.tech.hermes.management.domain.mode.ModeService;
```

Comment:
Please revert this change - I believe it's it is not necessary in this PR :)

Also true for:
- hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/GroupsEndpoint.java
- hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/MigrationsEndpoint.java
- hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/SchemaEndpoint.java
- hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/SubscriptionsEndpoint.java
- hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/TopicsEndpoint.java